src/containers/home/Home.js

```diff
@@ -0,0 +1,44 @@
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { Button } from "react-bootstrap";
+
+import * as userSelector from "../../store/selectores/user";
+import { useAuth } from "../auth/AuthProvider";
+import { ROLES } from "../../helpers/auth";
+import AuthorizedWithMessage from "../auth/authorizationWrappers/AuthorizedWithMessage";
+
+const Home = () => {
+  const { signOut } = useAuth();
+  const userName = useSelector(userSelector.userName);
+
+  const signOutUser = async () => {
+    signOut();
+  };
+
+  return (
+    <div data-testid="component-home" className="home-wrapper">
```

Comment:
test element id - https://empyreallogistics.atlassian.net/wiki/spaces/E/pages/153518081/Prerequisites+and+Guidelines#Test-Elements

Reply:
Fixed

Reply:
test elements id should be updated

home-sign-out -> home-sign_out

For reference - https://empyreallogistics.atlassian.net/wiki/spaces/E/pages/153518081/Prerequisites+and+Guidelines#Test-Elements

Reply:
Fixed